src/clusterfuzz/_internal/base/utils.py

```diff
@@ -124,6 +124,31 @@ def fetch_url(url):
   return response.text
 
 
+@retry.wrap(
+    retries=URL_REQUEST_RETRIES,
+    delay=URL_REQUEST_FAIL_WAIT,
+    function='base.utils.post_url')
+def post_url(url: str, data: str, headers: dict) -> str:
+  """Post the provided data and headers to the provided url.
+  The request is retried `URL_REQUEST_RETRIES` times.
+  To avoid blocking the application, a post timeout is applied using the
+  environment variable `URL_BLOCKING_OPERATIONS_TIMEOUT`.
+  Args:
+    url: the url to post to.
+    data: request data.
+    headers: request headers.
+  Returns:
+    the contents of the response, in unicode.
+  Raises:
+    raises an `HTTPError`, if one occurred."""
+  operations_timeout = environment.get_value('URL_BLOCKING_OPERATIONS_TIMEOUT')
```

Comment:
maybe let's use a sane value here like 60. 10 minutes seems too long to wait.

Reply:
Fixed